nodebuilder/das/config.go

```diff
@@ -23,6 +24,7 @@ func DefaultConfig(tp node.Type) Config {
 	switch tp {
 	case node.Light:
 		cfg.SampleTimeout = modp2p.BlockTime * time.Duration(cfg.ConcurrencyLimit)
+		cfg.SamplingWindow = light.Window.Duration()
```

Comment:
Would it be possible to make DASer config field SamplingWindow to be type of pruner.AvailabilityWindow? It will ensure type safety and remove the need for conversions